components/Shared/AccountCard/index.jsx

```diff
@@ -49,7 +52,9 @@ const AccountCard = forwardRef(
         <Box color='card.account.color'>
           <BigTitle>{alias}</BigTitle>
           <Box display='flex' alignContent='center'>
-            <Text margin={0}>{truncate(address)}</Text>
+            <AccountCardAddress fontWeight={1} fontSize={5} margin={0}>
```

Comment:
Just change to `<Title ...props />` - no reason to save an extra variable for this.

Reply:
My thinking: Since this is explicitly declaring an alternative `fontSize` prop to the standard `Label` component, using a different component name may indicate "why" (because it's not a formal `Label` component, but a unique variant of it).

But if that doesn't make sense to you I'll update it in a moment :)

Reply:
Then import it differently:

`import { Title as AccountAddressText } from '`

Reply:
@Schwartz10 Done! Thanks for the tip.